projects/briey/memtest/src/main.c

```diff
@@ -1,19 +1,20 @@
 #include <stdint.h>
 #include <stdlib.h>
+#include <stdio.h>
 #include <briey.h>
 
-#define PATTERN       (0b10101010u)
+#define PATTERN       (0x55555555U) // Repeating 0b0101...
```

Comment:
Could we go back to writing 1 byte at a time please? That would be a more comprehensive test in making sure cache readback isn't broken by the changes

Reply:
From offline discussion: leaving it as word transfers, adding todo for adding compile flag to test byte transfers.